src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -747,13 +945,14 @@ def forward(
 
         if not return_dict:
             head_outputs = (sequence_output, pooled_output) if pooled_output is not None else (sequence_output,)
-            return head_outputs + encoder_outputs[1:]
+            return head_outputs + encoder_outputs[1:] + embedding_output[1:]
 
-        return BaseModelOutputWithPooling(
+        return BaseModelOutputWithPoolingAndIntermediateActivations(
             last_hidden_state=sequence_output,
             pooler_output=pooled_output,
             hidden_states=encoder_outputs.hidden_states,
             attentions=encoder_outputs.attentions,
+            intermediate_activations=embedding_output.intermediate_activations,
```

Comment:
This is a breaking change, as users who are currently using `DPTModel` expect 4 keys in the output class => this will now be 5.

Reply:
Only if those are all not None however.